lib/features/login/data/model/oidc_configuration_cache.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:equatable/equatable.dart';
+import 'package:hive/hive.dart';
+import 'package:tmail_ui_user/features/caching/utils/caching_constants.dart';
+
+part 'oidc_configuration_cache.g.dart';
+
+@HiveType(typeId: CachingConstants.OIDC_CONFIGURATION_HIVE_CACHE_IDENTIFY)
+class OidcConfigurationCache extends HiveObject with EquatableMixin {
```

Comment:
why we need cache that?

Reply:
got it, but please change the name to `AuthorityOIDCCache`

Reply:
I switched to using `SharedPreference` to store `authority`